.cache-codegen.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
what is this tool?

Reply:
https://github.com/guanbinrui/api-cache-codegen

Reply:
if it is simple enough (e.g. download file from the internet), you can add it directly in `packages/scripts`, like `sync-languages` we have now.